app/views/notifier/_user_message_table.html.erb

```diff
@@ -0,0 +1,18 @@
+<table style="font-size: 15px; font-style: italic; margin: 15px; background-color: #eee; width: 520px">
```

Comment:
What I was trying to convey was that this partial template is for messages (within the email message) that are written by another user. So it's for direct messages, changeset comments, diary comments and note comments. It's the box with the avatar you can see in the screenshot above. Not all notification messages have it -- the password reset email doesn't for instance. 

But words like "message" and "body" quickly get confusing I agree because we're talking about a message body (the user's message) within another message body (the email itself).

I've renamed it to `_message_body`.